packages/nc-gui-v2/package.json

```diff
@@ -10,7 +10,8 @@
     "test:ui": "vitest -c test/vite.config.ts --ui",
     "coverage": "vitest -c test/vite.config.ts run --coverage",
     "build:copy": "npm run generate; rm -rf ../nc-lib-gui-v2/lib/dist/; rsync -rvzh ./dist/ ../nc-lib-gui-v2/lib/dist/",
-    "build:copy:publish": "npm run generate; rm -rf ../nc-lib-gui-v2/lib/dist/; rsync -rvzh ./dist/ ../nc-lib-gui-v2/lib/dist/; npm publish ../nc-lib-gui-v2"
+    "build:copy:publish": "npm run generate; rm -rf ../nc-lib-gui-v2/lib/dist/; rsync -rvzh ./dist/ ../nc-lib-gui-v2/lib/dist/; npm publish ../nc-lib-gui-v2",
+    "postinstall": "node scripts/updateNuxtRouting.js"
```

Comment:
Shouldn't this be postbuild? If I install in a fresh project there's no build output to modify, right?

Reply:
It's updating in nuxt package

Reply:
ah okay, I thought it's modifying the output 😄